tailcfg/tailcfg.go

```diff
@@ -638,6 +639,11 @@ type MapResponse struct {
 	// PeersRemoved are the NodeIDs that are no longer in the peer list.
 	PeersRemoved []NodeID `json:",omitempty"`
 
+	// PeerSeenChange contains information on how to update peers' LastSeen
+	// times. If the value is false, the peer is gone. If the value is true,
+	// the LastSeen time is now. Absent means unchanged.
+	PeerSeenChange map[NodeID]bool `json:",omitempty"`
```

Comment:
If we're really interested in squeezing bytes out, I think this be more efficient on the wire:

```go
SeenNow []NodeID `json:",omitempty"`
Unseen    []NodeID `json:",omitempty"`
```

Reply:
Compression will handle some redundant trues and falses fine. The big win is being able to avoid sending the whole node updates at all. I'd slightly prefer a single field.

Reply:
OK